src/JabbR-Core/wwwroot/Chat.js

```diff
@@ -413,7 +412,15 @@
         }
     };
 
+    //var hasBeenCalled = false;
     chat.client.roomLoaded = function (roomInfo) {
+        //if (!hasBeenCalled) {
```

Comment:
if this was something that was left out from old Jabbr go ahead and add an issue here #142 so we don't forget